code/game/gamemodes/cult/runes.dm

```diff
@@ -353,7 +353,7 @@ var/list/teleport_runes = list()
 	var/turf/T = get_turf(src)
 
 	for(var/mob/living/M in T.contents)
-		if(!iscultist(M) && !ismindshielded(M) && ishuman(M))
+		if(!iscultist(M) && !ismindshielded(M) && !isgolem(M) && !isanimal(M) && ishuman(M))
```

Comment:
can't you just typecast the `for` loop to check for humans? Since nothing except humans will pass this check

Reply:
Humans count for the mob not the speicies..and for some REASON golems are passing this check..as are penguins...

Reply:
oh right. Still, typecasting would fix your penguin issue

Reply:
all the is-calls (ishuman, isgolem) are ACTUALLY shorthand for a type check. This is pretty much me nuking from orbit.